templates/mod/ssl.conf.erb

```diff
@@ -10,6 +10,9 @@
   SSLPassPhraseDialog  builtin
   SSLSessionCache "shmcb:<%= @session_cache %>"
   SSLSessionCacheTimeout 300
+<% if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
+  SSLStaplingCache "shmcb:<%= @stapling_cache %>"
```

Comment:
And your correct as far as i know.

if ssl stapling cache directive set without ssl stapling on, apache would
fail to start.

Changes look ok. Seems fair enough.

On 10 February 2015 at 10:41, Matthias Viehweger <notifications@github.com>
wrote:

> In templates/mod/ssl.conf.erb
> <https://github.com/puppetlabs/puppetlabs-apache/pull/907#discussion_r24396247>
> :
>
> > @@ -10,6 +10,9 @@
> >    SSLPassPhraseDialog  builtin
> >    SSLSessionCache "shmcb:<%= @session_cache %>"
> >    SSLSessionCacheTimeout 300
> > +<% if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
> > +  SSLStaplingCache "shmcb:<%= @stapling_cache %>"
>
> I may be wrong, but shouldn't the SSLStaplingCache only be set if
> ssl_stapling is activated?
>
> I propose this conditional:
>
> if @ssl_stapling && scope.function_versioncmp([@apache_version, '2.4']) >= 0
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetlabs-apache/pull/907/files#r24396247>
> .
>